src/neo-vm/ExecutionEngine.cs

```diff
@@ -1303,11 +1303,17 @@ private bool HandleException()
             return false;
         }
 
-        public ExecutionContext LoadClonedContext(int initialPosition)
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public ExecutionContext LoadClonedContext(int initialPosition, int rvcount = 0)
+        {
+            return LoadClonedContext(CurrentContext, initialPosition, rvcount);
+        }
+
+        public ExecutionContext LoadClonedContext(ExecutionContext context, int initialPosition, int rvcount = 0)
```

Comment:
Can we remove this and use `LoadContext()`?

Reply:
Yes, If we change `Clone` to public